### ISSUE TITLE
It me again!

### ISSUE THREAD

**Issue by hairypaulsack** [prius0304/WarThunder-Data-Extractor#1]
So I tried using the old version and I ran into an error, when I finally got it debugged "Input string was not in a correct" I realized you had already fixed the issue in this second repository. Why have two git? Why not update the old one?

Anyway, I can not get this one to work. When I debug in vs2017 disable_debug() must be disabled, because I only see debug information when running outside of vs in /bin/wt.exe

When I do do this, state works but indicators comes back as an error.

I go into sln for warthunder.dll and I see that you prefix '/' in front of indicators. If your URL is set to http://localhost:8111/ then why have leading '/' for indicators? that would cause //... 

Not only this, but you have logic that if '/indicators' doesn't work, try 'indicators' correct?

Lastly, you have indicators spelled 'indicatorss' which is why bin/wt.exe was telling me indicators failed 404. I fixed the spelling error (which for some reason I think wasn't an accident???) and left the one /indicators and just removed the 'ss' to 's'.

404 error gone. clicking get real data returns nothing. 

All that aside, I am real big noob in programming. So I am curious how the process works. The process table is only meant to be used in IDE dev environment correct? No passing of parameters in cmd etc?

Also, can the .dll be used outside of vs? Like can I do dllcall from autohotkey to call the functions from the dll and used the data inside ahk or autoit? [dllcall](https://lexikos.github.io/v1/docs/commands/DllCall.htm)

**Comment by prius0304** [comment]
I'm truly sorry that I have no time to update this git again. And we gave up the WarThunder platform for some reasons, so I didn't notice these errors.
This program is baiscally a web crawler. It's much easiler to implement with python. You can download it and modify it without my permision.
The .dll can run without vs of course.

**Comment by hairypaulsack** [comment]
Yeah I was able to get it going on python with bs4 and some others, but I'm still curious. I was just wondering if I could access the functions similar to how windows 7 picture viewer is called with 

`rundll32.exe "C:\Program Files (x86)\Windows Photo Viewer\PhotoViewer.dll", ImageView_Fullscreen %1`
